aggsender/validator/validate_certificate.go

```diff
@@ -119,6 +112,15 @@ func (a *CertificateValidator) ValidateCertificate(ctx context.Context, params t
 		return fmt.Errorf("failed to verify claim proofs: %w", err)
```

Comment:
Should the `verifyClaimProofs` also be a part of `VerifyCertificate` function (it can be declared in the common flow for example), WDYT?